README.md

```diff
@@ -102,6 +110,8 @@ $CONFIG = array (
 'overwriteprotocol' => 'https',
 ```
 
-**Note:** If necessary, restart Nextcloud to clear the APCu cache for the config file.
-
-**Note:** You can use the above `Mapper` method to map any arbitrary user attribute in Keycloak to output with standard userdata, allowing use of arbitrary fields for `id`, etc.
+**Note:** 
+- If necessary, restart Nextcloud to clear the APCu cache for the config file.
+- You can use the above `Mapper` method to map any arbitrary user attribute in Keycloak to output with standard userdata, allowing use of arbitrary fields for `id`, etc.
+- Groups are currently not supported
+- Interferring with Nextcloud Password App
```

Comment:
Can you remove this last line. Let's not add issues to documentation unless we're sure they can't be fixed.

Reply:
Yes i will do :+1: